lib/service_variables.rb

```diff
@@ -68,7 +73,7 @@ def string_option(name, default: nil, enum: nil)
     end
 
     define_singleton_method "#{name}=" do |value|
-      raise InvalidValueError, "Only #{enum} values are allowed." if enum && !enum.include?(value)
+      raise InvalidValueError, "Only #{enum} values are allowed." unless enum&.include?(value)
```

Comment:
if you don't specify an enum, this will always throw wont it?

Reply:
Yea, fixed, weird rubocop changed this automatically